modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFClaim.testcase

```diff
@@ -190,23 +221,69 @@ definition {
 				thirdInvoice = "Document_1.png");
 		}
 
-		task ("And I go back to the MDF listing page") {
-			PRMUtils.gotoPRM();
+		task ("And the Partner user is logged in and goes to MDF listing page") {
```

Comment:
Hi @marinaasant0s, could you please rename this task to "And the Partner goes to MDF listing page"? Thanks!